### PR TITLE
Ss tf 1990 symbols vision os aab

### PR DESCRIPTION
**Pull request by simon-sarrafi**
### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/testfairy/app-distribution/symbols-file.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+After a successful upload via the API, the response will include a URL to download your Symbols file.
+
+Here’s an example of the relevant portion of the API response:
+
+```json
+{
+  "status": "ok",
+  ...
+  "symbols_download_url": "https://mobile.testfairy.com/api/xxx/projects/xxx/builds/xxx/symbols/download/",
```
Comment (kristofmuhi) [inline docs/testfairy/app-distribution/symbols-file.md:55]: doesnt the download url contains the file extension?

File: docs/testfairy/app-distribution/symbols-file.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  ...
+}
+```
+
+## Accessing Symbols File Through the Dashboard
+
+To download your symbols file through the Sauce Labs dashboard, follow these steps:
```
Comment (kristofmuhi) [inline docs/testfairy/app-distribution/symbols-file.md:62]: Lets call out here Sauce Labs (TestFairy)

File: docs/testfairy/app-distribution/symbols-file.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+1. **Log In to Your Dashboard**
+
+   Log in to your [Sauce Labs dashboard](https://saucelabs.com).
+
+2. **Navigate to Uploaded Apps**
+
+   From the sidebar menu go to Mobile Testing and to the section where your uploaded apps are listed. Find the app that has the symbols file attached to it.
```
Comment (kristofmuhi) [inline docs/testfairy/app-distribution/symbols-file.md:70]: LEts replace this with just navigating to mobile.saucelabs.com or to your dedicated TestFairy private instance

**Review by kristofmuhi — CHANGES_REQUESTED** [review]

File: docs/testfairy/app-distribution/symbols-file.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+- **Attachment:** Include the mapping file `symbols_file` as part of your app submission. For instance:
+
+Android:
+
+```bash
+curl https://upload.testfairy.com/api/upload \
+ -F api_key='your_api_key' \
+ -F file=@sample.apk \
+ -F symbols_file=@mappings.txt
```
[the remark this answers is not in the document]
Comment (simon-sarrafi) [reply docs/testfairy/app-distribution/symbols-file.md:33]: Yes  always text file for Android

File: docs/testfairy/app-distribution/symbols-file.md (hunk 1 above)
Comment (simon-sarrafi) [reply docs/testfairy/app-distribution/symbols-file.md:55]: No! the download url is the api endpoint which will call the actual aws url. This url remain the same for all platforms' files

File: docs/testfairy/app-distribution/symbols-file.md (hunk 3 above)
Comment (simon-sarrafi) [reply docs/testfairy/app-distribution/symbols-file.md:70]: At the 2nd point we are already there, so I would just say, get the app from your list.

**Review by kristofmuhi — APPROVED** [review]

**Pull request merged**